_rules/id-value-unique-3ea0c8.md

```diff
@@ -60,7 +60,7 @@ There is only one `id` within the document context.
 
 #### Passed Example 2
 
-All `id`s are unique within the document context.
+All `id` are unique within the document context.
```

Comment:
```suggestion
All `id` attributes are unique within the document context.
```

This is in keeping with #834 (preferred format examples) and #946 (test case description example) to identify code as an element or attribute.

Reply:
Good point.